pkg/arvo/sys/vane/eyre.hoon

```diff
@@ -2868,6 +2887,33 @@
     ==
   ::
       %~2023.4.11
+  ::
+  ::  Prior to this desks were not part of events.channel.
+  ::  When serializing we used to rely on the desk stored in
+  ::  subscriptions.channel, but this state is deleted when we clog.
+  ::  This migration adds the desk to events.channel, but we can not
+  ::  scry in +load to populate the desks in the old events,
+  ::  so we just kick all subscriptions on all channels.
+  ::
+    =;  new-channel-sessions
+      %=  $
+        -.old                                   %~2023.4.19
+        session.channel-state.server-state.old  new-channel-sessions
+      ==
+    %-  ~(run by session.channel-state.server-state.old)
+    |=  c=channel-3
+    =;  new-events
+      c(events new-events, unacked ~, subscriptions ~)
+    =|  events=(qeu [id=@ud request-id=@ud =channel-event])
+    =/  l  ~(tap in ~(key by subscriptions.c))
+    |-
+    ?~  l  events
+    %=  $
+      l          t.l
+      next-id.c  +(next-id.c)
+      events     (~(put to events) [next-id.c i.l %kick ~])
+    ==
+      %~2023.4.19
```

Comment:
Style nit: this should have a comment separate line before it.

Reply:
Done